src/writer.jl

```diff
@@ -161,11 +161,25 @@ function write(path::AbstractString, obj; force=false)
 
     # Close .shp file
     close(io)
-    
+
     # Write .shx file
     index_handle = IndexHandle(header, shx_indices)
     Base.write(paths.shx, index_handle)
 
+    # Write .prj file
+    crs = try
+        GI.crs(geoms)
+    catch
+        nothing
+    end
+    if !isnothing(crs)
+        try
+            wkt = convert(GeoFormatTypes.ESRIWellKnownText, crs)
+            Base.write(paths.prj, wkt.val)
+        catch
+            @warn "Could not write .prj file.  ArchGDAL may need to be loaded."
```

Comment:
If I understand correctly, ArchGDAL owns/pirates the `Base.convert` methods for types in GeoFormatTypes.  Is this a correct/acceptable warning?